insights/parsers/systemd/config.py

```diff
@@ -135,9 +138,9 @@ def parse_systemd_ini(content):
         dict_section = {}
         for option in Config.options(section):
             value = Config.get(section, option).splitlines()
-            value = map(lambda s: s.rstrip("\\").strip(), value)
-            value = filter(lambda s: bool(s), value)
-            dict_section[option] = "\n".join(value)
+            value = list(unsplit_lines(value))
+            """ If the len of value is 1, then set the value as string"""
```

Comment:
For single-line comment, better use hash (#) style `# comment`
And there is a typo: as -> is